ddtrace/internal/ci_visibility/encoder.py

```diff
@@ -32,7 +32,8 @@ class CIVisibilityEncoderV01(BufferedEncoder):
     ALLOWED_METADATA_KEYS = ("language", "library_version", "runtime-id", "env")
     PAYLOAD_FORMAT_VERSION = 1
     TEST_SUITE_EVENT_VERSION = 1
-    TEST_EVENT_VERSION = 2
+    # Change to 1 to allow unittest tests without module or session spans
+    TEST_EVENT_VERSION = 1
```

Comment:
How does this affect existing uses of `TEST_EVENT_VERSION`?

Reply:
Our `pytest` plugin is meant to use `TEST_EVENT_VERSION` `2`, so I think we should either make versioned instances of the encoder and have `pytest` use v2 and `unittest` use v1, or just make sure `unittest` does support version `2`.

Reply:
An update on this: `TEST_EVENT_VERSION` is now set to 2 (just like before) if the `DD_CIVISIBILITY_UNITTEST_BETA_ENABLED` is not True / or not set. This should not affect existing uses of `dd-trace-py` if the env var unittest is not set.

It is planned to rollback to `TEST_EVENT_VERSION = 2` with the full PR soon